_docs/_latest/setup/kubernetes.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Deploy Forseti Security on Kubernetes
+order: 004
+---
+
+# {{ page.title }}
+
+This guide explains how to setup Forseti on Kubernetes.
+
+Note: The current work here is considered not yet ready for production use.
+Instead, it is to be a starting point to iterate and improve upon before
+releasing a smoother installation approach for general usage, possibly by
+using pre-built images stored on a public repo such as DockerHub.
+
+---
+
+## Pre-Requistes
+
+Install Forseti server and client using the [standard
+installer]({% link _docs/latest/setup/install.md %}) or the [terraform
+installer](https://github.com/terraform-google-modules/terraform-google-forseti).
+This is a quick way to install the service accounts, Cloud SQL and GCS buckets.
+The VM's may be deleted or shutdown after the install as they will be replaced
+by the k8s solution.
+
+Alternatively follow the [manual setup instructions]({% link _docs/latest/setup/manual.md %})
+for service accounts, Cloud SQL and GCS buckets.
+
+## Build Docker Image
+
+There are various ways to build the docker image, using git clone or
+a personal fork.
+
+Note: [cloudbuild.yaml](https://github.com/GoogleCloudPlatform/forseti-security/blob/docker-poc/install/docker/cloudbuild.yaml)
```

Comment:
Pls remove this paragraph. The cloud build cache step has been made fail safe.

Reply:
Done.